app/models/pawn.rb

```diff
@@ -1,2 +1,64 @@
 class Pawn < ChessPiece
+  INITIAL_MOVE_FACTOR = 2
+  POST_INITIAL_MOVE_FACTOR = 1
+
+  def valid_move?(destination_x, destination_y)
+    return false unless forward_move?(destination_y)
+    if vertical_move?(destination_x, destination_y)
+      valid_vertical_move?(destination_x, destination_y)
+    else
+      valid_capture?(destination_x, destination_y)
+    end
```

Comment:
looks really good. could remove the else and end with `valid_capture?(destination_x, destination_y`). And add a return before `valid_vertical_move`